src/App.js

```diff
@@ -1,16 +1,78 @@
 import React from 'react';
+import './styles/app.scss';
+import Header from './components/TodoComponents/Header';
+import Decide from './components/TodoComponents/Decide';
+import Todos from './components/TodoComponents/Todos';
+import AddTodo from './components/TodoComponents/AddTodo';
+import VisibilityToggle from './components/TodoComponents/VisibilityToggle.js';
+
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
-  render() {
-    return (
-      <div>
-        <h2>Welcome to your Todo App!</h2>
-      </div>
-    );
-  }
+    constructor(props) {
+        super(props);
+        this.deleteTodos = this.deleteTodos.bind(this);
+        this.decideTodo = this.decideTodo.bind(this);
+        this.handleAddTodo = this.handleAddTodo.bind(this);
+        this.deleteTodo = this.deleteTodo.bind(this);
+        this.state = {
+            todos: []
+        }
+    }
+
+    deleteTodos() {
+        this.setState( () => ( { todos: [] } ) )
```

Comment:
It's cool that you used setState with a function, as we haven't covered that yet! You only really do this if you're updating data. The function can take in the previous state as a parameter